cmd/webhook/server/options.go

```diff
@@ -54,6 +56,7 @@ func NewWebhookServerOptions() *WebhookServerOptions {
 // AddFlags adds flags for a WebhookServerOptions to the specified FlagSet.
 func (s *WebhookServerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.IntVar(&s.HealthzServerBindPort, "healthz-server-bind-port", defaultHealthzServerPort, "The port on which to serve HTTP  /healthz endpoint")
```

Comment:
This is already overridden to 8081 in deployment, so it causes an error with the default for `ControllerManagerMetricsPort` 

https://github.com/kubernetes-sigs/service-catalog/blob/f83ebf922fd676cf75b5541d9980737da9f91203/charts/catalog/templates/webhook-deployment.yaml#L50-L51